<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestAppClass</class>
 <widget class="QMainWindow" name="TestAppClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>558</width>
    <height>408</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TestApp</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="labelKapow">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>291</width>
      <height>271</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="testapp.qrc">:/kapow.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonKapow">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>0</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>KAPOW</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelBoom">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>20</y>
      <width>331</width>
      <height>261</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="testapp.qrc">:/boom2.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxPickImg">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>---pick img---</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>KAPOW</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>BOOM</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButtonBoom">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>0</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>BOOM</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonClearScreen">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>0</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>clear screen</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>558</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="testapp.qrc"/>
 </resources>
 <connections/>
</ui>
